tests/nightly/unity/task.yaml

```diff
@@ -25,13 +25,12 @@ disabled_restore: |
 
 execute: |
     echo "Given a unity snap is installed"
-    snap install ubuntu-clock-app
+    snap install sensors-unity
```

Comment:
Should we rather use a snap that's published by us so that it does not randomly go away again?

Reply:
Also I wonder how this will skew the metrics for that particular snap.

Reply:
all snaps in spread tests don't show up in metrics due to the SNAPPY_TESTING env var we define in spread.yaml